lib/PuppeteerSharp/Browser.cs

```diff
@@ -39,15 +39,21 @@ public class Browser : IDisposable
         /// Initializes a new instance of the <see cref="Browser"/> class.
         /// </summary>
         /// <param name="connection">The connection</param>
-        /// <param name="options">The browser options</param>
+        /// <param name="ignoreHTTPSErrors">The option to ignoreHTTPSErrors</param>
+        /// <param name="setDefaultViewport">The option to setDefaultViewport</param>
         /// <param name="process">The chrome process</param>
         /// <param name="closeCallBack">An async function called before closing</param>
-        public Browser(Connection connection, IBrowserOptions options, Process process, Func<Task> closeCallBack)
+        public Browser(
+            Connection connection,
+            bool ignoreHTTPSErrors,
+            bool setDefaultViewport,
+            Process process,
+            Func<Task> closeCallBack)
         {
             Process = process;
             Connection = connection;
-            IgnoreHTTPSErrors = options.IgnoreHTTPSErrors;
-            AppMode = options.AppMode;
+            IgnoreHTTPSErrors = ignoreHTTPSErrors;
+            AppMode = setDefaultViewport;
```

Comment:
In order to get closer to the pptr implementation, I’d remove the AppMode property and replace it with the SetDefaultViewport one.
This would also change how the target creates a page 
https://github.com/kblok/puppeteer-sharp/blob/6a16150d4bb6edee07ad1d5e7c39fcfb6e300715/lib/PuppeteerSharp/Target.cs#L96